nautobot/cloud/navigation.py

```diff
@@ -30,10 +30,26 @@
                             ),
                         ),
                     ),
+                    NavMenuItem(
+                        link="cloud:cloudservice_list",
+                        name="Cloud Services",
+                        weight=200,
+                        permissions=[
+                            "cloud.view_cloudservice",
+                        ],
+                        buttons=(
+                            NavMenuAddButton(
+                                link="cloud:cloudservice_add",
+                                permissions=[
+                                    "cloud.add_cloudservice",
+                                ],
+                            ),
+                        ),
+                    ),
                     NavMenuItem(
                         link="cloud:cloudtype_list",
                         name="Cloud Types",
-                        weight=200,
+                        weight=250,
```

Comment:
Let's stick to multiples of 100 for the cloud menu items unless there's a strong need not to. See also https://github.com/nautobot/nautobot/pull/5916/files#r1674067073